module/input/SensorFilter/src/MotionModel.hpp

```diff
@@ -181,32 +170,30 @@ namespace module::input {
             return newState;
         }
 
-        Eigen::Matrix<Scalar, 3, 1> predict(const Eigen::Matrix<Scalar, size, 1>& state,
-                                            const MeasurementType::ACCELEROMETER&) {
-            // Extract our rotation quaternion
-            const Eigen::Matrix<Scalar, 3, 3> Rtw = StateVec(state).Rwt.inverse().toRotationMatrix();
-
-            // Make a world gravity vector and rotate it into torso space
-            // Where is world gravity with respect to robot orientation?
-            // Multiplying a matrix with (0, 0, G) is equivalent to taking the
-            // third column of the matrix and multiplying it by G
-            return Rtw.template rightCols<1>() * G;
+        Eigen::Matrix<Scalar, 3, 1> predict(const StateVec& state, const MeasurementType::ACCELEROMETER&) {
+
+            // Rotate world gravity vector into torso space using quaternion conjugation
+            // p' = q * p * q.conjugate()
```

Comment:
This line is indeed a quaternion conjugation, but I don't see how it relates to the previous line. Like, what are the variable substiutions? At the moment, I am thinking that it's saying that `p` is the world gravity vector and conjugating with `q` rotates it into torso space? That conclusion took me a couple of minutes, and I am dubious that it is correct.

Reply:
This makes perfect sense

Reply:
I am glad you think so. I don't follow it.